cookbook/email/dev_environment.rst

```diff
@@ -140,11 +140,11 @@ by adding the ``delivery_whitelist`` option:
             delivery_whitelist:
                # all email addresses matching this regex will *not* be
                # redirected to dev@example.com
-               - "/@specialdomain.com$/"
+               - "/@specialdomain[.]com$/"
```

Comment:
@gonzalovilaseca although I can't provide a solid reasoning for this, I agree with @xabbuh here. When escaping just one character, I find it "better" to use the backslash instead of the other notation:

```php
- "/@specialdomain[.]com$/"

- "/@specialdomain\.com$/"
```